pybossa/jobs.py

```diff
@@ -958,11 +958,19 @@ def export_tasks(current_user_email_addr, short_name,
             with export_fn(project, ty, expanded, filters, disclose_gold) as fp:
                 filename = fp.filename
                 content = fp.read()
-            data = project.info
-            bucket_name = current_app.config.get('EXPORT_BUCKET')
-            max_size = current_app.config.get('EXPORT_MAX_SIZE', float('Inf'))
 
-            if len(content) > max_size and bucket_name:
+            bucket_name = current_app.config.get('EXPORT_BUCKET')
+            max_email_size = current_app.config.get('EXPORT_MAX_EMAIL_SIZE', float('Inf'))
+            max_s3_upload_size = current_app.config.get('EXPORT_MAX_UPLOAD_SIZE', float('Inf'))
+
+            if len(content) > max_s3_upload_size:
+                current_app.logger.info(f"task export exceeded max size. Project ID: {project.id}, Size: {len(content)}")
```

Comment:
It is recommended to avoid f-string in logging due to performance issue. refer [this](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-format-interpolation.html) and [this](https://stackoverflow.com/questions/11955787/idiomatic-python-logging-format-string-args-list-vs-inline-string-formatting)